handout/architect/10_ways_to_misuse_angular.md

```diff
@@ -0,0 +1,103 @@
+# Ten Ways to Misuse Angular
+
+1. Ignoring a component’s host element
+    - No need to wrap templates in a `<div>`
+    - Use `:host` selector
+    - Use `@HostBinding` and `@HostListener`
+    - Subtle differences host metadata vs. `@Host*` decorators
+1. Avoiding observables
+    - Leverage observable operators
+    - Merge and transform data in streams rather than direct data mutation
+    - Don’t prematurely subscribe
+    - Leverage `async` pipe where appropriate
```

Comment:
"where appropriate" is one of those phrases I really hate as a reader - always leaves me asking, when *is* it appropriate??